lightning/src/ln/peers/handler.rs

```diff
@@ -214,6 +221,30 @@ struct PeerHolder<Descriptor: SocketDescriptor, TransportImpl: ITransport> {
 	node_id_to_descriptor: HashMap<PublicKey, Descriptor>,
 }
 
+impl<Descriptor: SocketDescriptor, TransportImpl: ITransport> PeerHolder<Descriptor, TransportImpl> {
+	fn initialized_peer_by_node_id(&mut self, node_id: &PublicKey) -> Option<(Descriptor, &mut Peer<TransportImpl>)> {
```

Comment:
Agreed. This seems obvious to me since I've been in the code, but may not be to a new reader.

Done: `review: Small nits, documentation, and fixups`